SDK/AppCenter/Microsoft.AppCenter.UWP/Utils/DeviceInformationHelper.cs

```diff
@@ -33,18 +31,12 @@ public override async Task<Ingestion.Models.Device> GetDeviceInformationAsync()
             return await base.GetDeviceInformationAsync().ConfigureAwait(false);
         }
 
-        internal static void SetCountryCode(string country)
-        {
-            _country = country;
-            InformationInvalidated?.Invoke(null, EventArgs.Empty);
-        }
-
         public DeviceInformationHelper()
         {
             _screenSizeProvider = _screenSizeProviderFactory.CreateScreenSizeProvider();
             _screenSizeProvider.ScreenSizeChanged += (sender, e) =>
             {
-                InformationInvalidated?.Invoke(sender, e);
+                InvalidateInformation();
```

Comment:
Can this not cause a null reference exception? I think there was a reason for doing it the other way